OCRManager/DigitHelper.cs

```diff
@@ -66,10 +45,8 @@ public static IEnumerable<int> GetSimilarDigits(string input)
         {
             foreach (var candidateDigit in ValidDigits)
             {
-                if (1 == XORStrings(input, candidateDigit).Count(ch => ch == '1'))
-                {
+                if (input.Where((inputChar, index) => inputChar != candidateDigit[index]).Count() == 1)
```

Comment:
Easy and simple!
I clearly overthought this after realizing that `.Intersect()` and `.Except()` didn't work due to the way digits were being stored.

Reply:
That was exactly my initial thought process @ Intersect & Except!